test/redis.conf

```diff
@@ -0,0 +1,32 @@
+# Redis2 testing configuration file.
```

Comment:
do we need all of these variables? Can it possibly just be the `requirepass` field?

Reply:
I'll try to limit yo just `requirepass` and `port`

Reply:
Was able to limit the redis.conf to a minimalistic conf.